packages/instant/tslint.json

```diff
@@ -2,6 +2,7 @@
     "extends": ["@0x/tslint-config"],
     "rules": {
         "custom-no-magic-numbers": false,
-        "semicolon": [true, "always", "ignore-bound-class-methods"]
+        "semicolon": [true, "always", "ignore-bound-class-methods"],
+        "max-classes-per-file": false
```

Comment:
I refactored the `EthAmountRow`component to be a class instead of a function, and tslint started complaining.  This felt like a silly distinction, so thought we should turn off the rule -- open to thoughts on this though